mappings/net/minecraft/world/chunk/light/ChunkLightProvider.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS cer net/minecraft/world/chunk/light/ChunkLightProvider
+	FIELD a chunkProvider Lbro;
+	FIELD b type Lbca;
+	FIELD c lightStorage Lcet;
+	FIELD d DIRECTIONS [Ley;
+	FIELD f srcMutablePos Let$a;
+	FIELD g destMutablePos Let$a;
+	FIELD h mutablePosGetLightBlockedBetween Let$a;
```

Comment:
This may be a bit excessively verbose. 😄

Reply:
Any idea for a shorter name?

Reply:
While maybe not specific enough, could possibly be just `mutablePos`? It is only used in `getLightBlockedBetween`, but it could be used more generically.

Reply:
Looking at the way it's done, mojang isn't reusing them between methods at all. The 2 fields above that one are restricted to use in a single method too.